parser/instructions/parse.go

```diff
@@ -10,8 +10,8 @@ import (
 
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/strslice"
-	"github.com/docker/dockerfile/command"
 	"github.com/docker/dockerfile/parser"
+	"github.com/docker/dockerfile/parser/command"
 	"github.com/moby/buildkit/util/suggest"
```

Comment:
We should remove dependencies on buildkit itself. Looks like `github.com/moby/buildkit/util/suggest` is only use by the parser so suggest to move it here from BuildKit repo.

Reply:
There is also `github.com/moby/buildkit/util/stack` in `parser/errors.go`.